test/e2e/cli-install-uninstall/03-check-events-for-cleanup.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+- command: bash -c 'kubectl get -A events | grep -q "Execution of plan cleanup finished with status COMPLETE"'
```

Comment:
Events are resources as anything else, you can just assert on that resource and you don't need bash for it... I think we're already using it somewhere.

Reply:
Nice! Will use that instead 🎉